.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,31 @@
+name: shellcheck
+on:
+  pull_request:  
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+
+jobs:
+  shellcheck:
+    runs-on: gardenlinux-amd64
```

Comment:
we can probably just use the free github hosted runners (`runs-on: ubuntu-latest`) for this, shellcheck shouldn't need the additional performance of self hosted ones

Reply:
Unfortunately latest ubuntu image available for the GitHub runner is `20.04` and this has an older shellcheck.. but now that I am adding to sources.list anyway, I need to try again